### PR TITLE
O3-2904 Queue Module - queue should not have Queue Entries as a direc…

### PR DESCRIPTION
**Pull request by mseaton**
…t property

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/queue/model/Queue.java [hunk 1]
```diff
@@ -62,30 +63,9 @@ public class Queue extends BaseChangeableOpenmrsMetadata {
 	@JoinColumn(name = "status_concept_set", referencedColumnName = "concept_id")
 	private Concept statusConceptSet;
 	
-	@OneToMany(mappedBy = "queue", cascade = CascadeType.ALL, fetch = FetchType.LAZY)
-	private List<QueueEntry> queueEntries;
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/queue/model/Queue.java:66]: I'm a little confused by this part of the change. Does this mean that queues no longer have queue entries?

Comment (mseaton) [reply api/src/main/java/org/openmrs/module/queue/model/Queue.java:66]: @ibacher yes, basically.  It says that "Queue Entries" belong to a Queue, but a Queue does not directly have access to the entries that are associated with it.  Can you find another example of where it's been our practice to have `data` belong to `metadata` directly?  You can't get a `List<PatientProgram>` from a `Program` for example, even if one might think they want to do something like `program.getEnrollments()`

Comment (ibacher) [reply api/src/main/java/org/openmrs/module/queue/model/Queue.java:66]: Nope. It always bothered me that we had this direct linkage! In retrospect, I should've just read the ticket.

Comment (ojwanganto) [reply api/src/main/java/org/openmrs/module/queue/model/Queue.java:66]: The refactoring makes sense. Thanks @mseaton

**Pull request merged**